browser/js/topics/topic.html

```diff
@@ -9,12 +9,12 @@
           <!-- prerequisite topic title -->
           <a ui-sref="topic({ topicId: prereq.prerequisiteId })">{{ prereq.title }}</a>
```

Comment:
This file is pretty large. Consider refactoring by porting some of the HTML here into various directives.